app/settings/requires-elevation/admin/template.hbs

```diff
@@ -143,7 +143,7 @@
             </ul>
           {{else}}
             <button {{action 'showOtpRecoveryCodes'}} class="btn btn-primary" type="submit">
-              Show backup codes
+              Show Backup Codes
```

Comment:
@chasballew sure;

- `Start 2FA activation` is shown when you haven't ever touched 2FA. When you click that, we provide the QR code, prompt you for an OTP token, and show you the next button: `Enable 2FA`
- When you click `Enable 2FA`, we check the OTP token and then enforce 2FA on your account if it's valid.